java/sage/NetworkEncoderManager.java

```diff
@@ -64,7 +64,7 @@ public void detectCaptureDevices(CaptureDevice[] alreadyDetectedDevices)
       }
     }
 
-    if (Sage.getBoolean("network_encoder_discovery", false))
+    if (Sage.getBoolean("network_encoder_discovery", true))
```

Comment:
Can you just make this another commit please? I'm fine with this change; but I don't want it mixed into a commit for something totally unrelated.